zio-sbt-githubactions/src/main/scala/zio/sbt/githubactions/model.scala

```diff
@@ -175,17 +226,23 @@ object Step {
     name: String,
     id: Option[String] = None,
     uses: Option[ActionRef] = None,
-    condition: Option[Condition] = None,
-    parameters: Map[String, Json] = Map.empty,
+    `if`: Option[Condition] = None,
+    `with`: Option[ListMap[String, Json]] = None,
     run: Option[String] = None,
-    env: Map[String, String] = Map.empty
+    env: Option[ListMap[String, String]] = None
```

Comment:
Why not `env: ListMap[String, String] = ListMap.empty`?

Reply:
Optional keys are automatically left out of the json object. An empty map will be encoded as `{}`. I do not think zio-json has the option to exclude keys with empty objects yet.

Reply:
Perhaps in zio-json this could be fixed with a default value. Dropping the key when the value is the default value.

Reply:
I prefer to make it as simple as possible! So probably it should be fixed on zio-json side.